### PR TITLE
py-intervaltree: new package at 3.0.2

### PR DESCRIPTION
**Pull request by codeandkey**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-intervaltree/package.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+
+class PyIntervaltree(PythonPackage):
+    """Editable interval tree data structure for Python 2 and 3."""
+
+    homepage = "https://github.com/chaimleib/intervaltree"
+    url      = "https://github.com/chaimleib/intervaltree/archive/3.0.2.tar.gz"
+
+    version('3.0.2', sha256='e8ab75b66077f2e5fb85ac56cb6df834689edb048d38601d53d8867cce3b77d1')
+
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-intervaltree/package.py]:
```python
depends_on('py-setuptools', type='build')
```

File: var/spack/repos/builtin/packages/py-intervaltree/package.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    depends_on('py-sortedcontainers@2:3', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-intervaltree/package.py]:
```suggestion
    depends_on('py-sortedcontainers@2.0:2.999', type=('build', 'run'))
```
Version ranges in Spack are inclusive.

File: var/spack/repos/builtin/packages/py-intervaltree/package.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    depends_on('py-pytest', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-intervaltree/package.py]:
```suggestion
    depends_on('py-pytest', type='test')
```

Comment (codeandkey) [reply var/spack/repos/builtin/packages/py-intervaltree/package.py]: Interesting, I didn't know test types were in place! Updated.

File: var/spack/repos/builtin/packages/py-intervaltree/package.py (hunk 2 above)
Comment (codeandkey) [reply var/spack/repos/builtin/packages/py-intervaltree/package.py]: Updated.

File: var/spack/repos/builtin/packages/py-intervaltree/package.py (hunk 1 above)
Comment (codeandkey) [reply var/spack/repos/builtin/packages/py-intervaltree/package.py]: Added!

**Pull request merged**